python/lsst/sims/maf/plots/plotHandler.py

```diff
@@ -9,6 +9,29 @@
 
 __all__ = ['PlotHandler', 'BasePlotter']
 
+
+def none_min(x):
+    """
+    check for min in an array and ignore None values
+    """
+    newx = [val for val in x if val is not None]
+    if len(newx) > 0:
+        return min(newx)
+    else:
+        return 0.
+
+
+def none_max(x):
+    """
+    check for min in an array and ignore None values
+    """
+    newx = [val for val in x if val is not None]
+    if len(newx) > 0:
+        return max(newx)
+    else:
+        return 0.
```

Comment:
Do we want to be returning 0 instead of `None`?  This gets used for setting the bounds of plot axes.  What happens if those bounds are `None`?  I assume matplotlib will just choose something on its own, which is probably the behavior we want.